packages/nimble-components/src/select/specs/filterable-select-hld.md

```diff
@@ -98,18 +133,10 @@ There is also a desire to allow the [`ListOption` to contain complex content](ht
 
 It may be desireable to have other filter modes in the future, such as case sensitive, or even regular expressions. By making the new API an enum, we can easily add new modes as needed.
 
-#### Dynamic fetching of options
-
-We know that there is a use-case with the `Combobox` to dynamically fetch options from a server that match the pattern provided in the input field, and so it isn't a stretch that a client might want the same capability in the `Select`. However, this is currently accomplished through turning off the `Combobox` `autocomplete` mode, and essentially having the client provide a custom behavior.
-
-The `Select` presents its own challenges for providing a similar ability:
-
--   Should the `Combobox` and `Select` provide mirrored APIs for this? Currently, this doesn't seem possible in Angular as the `Combobox` relies on users accessing its `value` property from the `nativeElement` to use for the filter, in combination with listening to native `input` events. The `Select` would either need to work differently, or the `Combobox` would have to be updated.
--   Would this feature be enabled through another mode on the `filterMode` enum (i.e. a `dynamic` or `custom` mode), or is it orthogonal to the `filterMode` API?
--   Are there challenges in having the filter work against local options in addition to retrieving new ones?
-
 ## Alternative Implementations / Designs
 
 None
 
 ## Open Issues
+
+-   Does the `Combobox` need to present a mirrored API to the `Select` for dynmically loaded options (i.e. an `isLoading` attribute along with the accompanying visuals, and a `filter-input` event)
```

Comment:
I'm not sure the HLD is the right place to hash this out. I think we should probably split it into a separate work item and declare it currently out of scope by listing it as a future consideration.